tests/Main/Util/Util.fs

```diff
@@ -46,26 +46,89 @@ let ``1foo`` = 5
 
 open Testing
 
+module private RunResult =
+#if FABLE_COMPILER
+    let private isString value : bool =
+        Fable.Core.JsInterop.emitJsExpr
+            (value)
+            // literal string: `"foo"`   // string object: `new String("foo")`
+            "typeof $0 === \"string\" || $0 instanceof String"
```

Comment:
Is there a Fable way to test for string?

It's just pretty printing in unit tests...but still...

Reply:
You can do type testing against string (as in `x :? string`) and this will be translated to `typeof x === "string"`. Currently is not checking `x instanceof String` though, maybe we should do that 🤔

Reply:
Haven't though of type pattern matching. Good idea, thanks!

(But: doesn't work in function with generic parameter:
```
error FSHARP: This runtime coercion or type test from type
    'c
to
    string
involves an indeterminate type based on information prior to this program point. Runtime type tests are not allowed on some types. Further type annotations are needed. (code 8)
```

-> parameter must be of type `obj`. Here that's ok.
)


I don't think `instanceof String` matters in practice: `String` (with capital `S`) is basically never used.... And here it's definitely not required (even if: What would happen is: String wouldn't get wrapped in quotation marks...)